<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1850</width>
    <height>925</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QComboBox" name="leftBox"/>
          </item>
          <item>
           <widget class="QLabel" name="labelLeftPath">
            <property name="text">
             <string/>
            </property>
            <property name="margin">
             <number>3</number>
            </property>
            <property name="indent">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="Leftchange">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="LeftTree">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="Panel" name="leftPanel"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="LeftTable">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QGridLayout" name="gridLayout_2"/>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="leftInfo">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>178</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QComboBox" name="rightBox"/>
          </item>
          <item>
           <widget class="QLabel" name="labelRightPath">
            <property name="text">
             <string/>
            </property>
            <property name="margin">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="RightChange">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="RightTree">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="Panel" name="rightPanel"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="RightTable">
             <layout class="QGridLayout" name="gridLayout_4"/>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="rightInfo">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="viewButton">
          <property name="text">
           <string>F3 Просмотр</string>
          </property>
          <property name="shortcut">
           <string>F3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="copyButton">
          <property name="text">
           <string>F5 Копирование</string>
          </property>
          <property name="shortcut">
           <string>F5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="moveButton">
          <property name="text">
           <string>F6 Перемещение</string>
          </property>
          <property name="shortcut">
           <string>F6</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="createButton">
          <property name="text">
           <string>F7 Создать директорию</string>
          </property>
          <property name="shortcut">
           <string>F7</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="text">
           <string>F8 Удаление</string>
          </property>
          <property name="shortcut">
           <string>F8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sortButton">
          <property name="text">
           <string>F12 Сортировка</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exitButton">
          <property name="text">
           <string>AltF4 Выход</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Panel</class>
   <extends>QTreeView</extends>
   <header>panel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
